style.css

```diff
@@ -493,3 +498,282 @@ form ul li button:hover {
   background-color: #fff5e1;
   border: 1px solid #f55800;
 }
+
+/* -----------------------------css for desktop or bigger screens-------- */
+@media only screen and (min-width: 768px) {
+  main {
+    width: 100%;
+  }
+
+  .menu {
+    display: none;
+  }
+
```

Comment:
- [ ] As suggested by the previous reviewer, please fix the horizontal scroll bar as it doesn't make the user experience good.

<img width="959" alt="image" src="https://user-images.githubusercontent.com/52934128/214811621-d8652459-1c2a-4988-830f-37cf56e57ffd.png">